eng/pipelines/performance/templates/build-perf-sample-apps.yml

```diff
@@ -1,35 +1,21 @@
 parameters:
   osGroup: ''
-  osSubgroup: ''
-  archType: ''
-  buildConfig: ''
-  runtimeFlavor: ''
-  helixQueues: ''
-  targetRid: ''
+  flavor: 'mono' # Currently only used for Android Hello World app
   nameSuffix: ''
-  platform: ''
-  shouldContinueOnError: ''
-  rootFolder: ''
-  includeRootFolder: ''
-  displayName: ''
-  artifactName: ''
-  archiveExtension: ''
-  archiveType: ''
-  tarCompression: ''
   hybridGlobalization: False
 
 steps:
 # Build Android sample app
   - ${{ if eq(parameters.osGroup, 'android') }}:
-    - script: make run TARGET_ARCH=arm64 DEPLOY_AND_RUN=false
+    - script: make run TARGET_ARCH=arm64 DEPLOY_AND_RUN=false RUNTIME_FLAVOR=${{ parameters.flavor }}
       workingDirectory: $(Build.SourcesDirectory)/src/mono/sample/Android
-      displayName: Build HelloAndroid sample app
+      displayName: Build HelloAndroid sample app RUNTIME_FLAVOR=${{ parameters.flavor }}
     - template: /eng/pipelines/common/upload-artifact-step.yml
       parameters:
         rootFolder: $(Build.SourcesDirectory)/artifacts/bin/AndroidSampleApp/arm64/Release/android-arm64/Bundle/bin/HelloAndroid.apk
         includeRootFolder: true
-        displayName: Android Mono Artifacts
-        artifactName: AndroidMonoarm64
+        displayName: Android HelloWorld ${{ parameters.flavor }} Artifacts
+        artifactName: AndroidHelloWorldArm64${{ parameters.flavor }}
```

Comment:
nit:
```suggestion
        displayName: Android Sample App JIT ${{ parameters.flavor }}
        artifactName: AndroidHelloWorldArm64${{ parameters.flavor }}
```

For consistency with other samples the naming should match as much as possible (plus include some info about the configuration)